src/main/java/br/com/openenade/api/municipio/Municipio.java

```diff
@@ -0,0 +1,101 @@
+package br.com.openenade.api.municipio;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import br.com.openenade.api.estado.Estado;
+
+@Entity
+public class Municipio {
+
+    @Id
+    private Long codigo;
+
+    @ManyToOne(cascade = CascadeType.ALL)
```

Comment:
@lucasvictor3 see the refactor that I made into Estado and Regiao relationship, furthermore please see how I fix the Estado and Regiao Service implementation.